mmset.raw.html

```diff
@@ -3683,69 +3683,69 @@
 variable pair is officially called a "disjoint variable pair" in the Metamath
 specification.  We present the rule as three separate cases above for clarity.
 In the set theory database file, set.mm, we adopt the convention that at least
-one setvar variable always appears in a distinct variable pair, so these are
+one setvar variable always appear in a distinct variable pair, so these are
```

Comment:
What's wrong with "one setvar variable always appears"?

Reply:
I think subjonctive sounds more natural, but I guess it can be either way.

Reply:
(For example: "we adopt the convention that it be blue", not "that it is blue".)

Reply:
Benoît seems to be correct, from a grammar point of view.
Maybe native speakers can confirm whether this is natural for them?

Reply:
I'm pretty sure "appears" is grammatically correct, since if one removes "always" then it fits perfectly
![image](https://github.com/metamath/set.mm/assets/58114641/367b12a0-69ff-47a4-8a9e-a2d14a0e066e)

Reply:
I had to try with the full sentence, and other cases.
<img width="1473" alt="image" src="https://github.com/metamath/set.mm/assets/5831830/42e8f073-8423-4dc5-9e3f-ee977659b494">

Reply:
This native speaker suggests "at least one setvar variable should always appear in a distinct variable pair" (word "should" added, "appear" in place of "appears"). Or some other rewording. I wouldn't rely too heavily on the subjunctive in English. In this sentence, "appear" without "should" doesn't really read to me as a subjunctive (at least, not on first reading) and my general understanding is that the subjunctive in English is much less used than in romance languages.

Reply:
Done in https://github.com/metamath/set.mm/pull/3786/commits/346c93b756c19bb58432ae0ebec5995a30660992.